backend/controllers/whereabouts/scheduledMoves.ts

```diff
@@ -23,26 +26,21 @@ type ScheduledMovementDetails = {
 }
 
 export default ({ prisonApi, offenderSearchApi }) => {
-  const renderTemplate = (
-    res,
-    { date, agencyDetails, movementReasons, courtEvents, releaseEvents, transferEvents, movementReason }
-  ) =>
+  const renderTemplate = (res, { date, agencyDetails, courtEvents, releaseEvents, transferEvents, scheduledType }) =>
     res.render('whereabouts/scheduledMoves.njk', {
       dateForTitle: moment(date, 'DD/MM/YYYY').format('D MMMM YYYY'),
       agencyDescription: agencyDetails.description,
       formValues: {
         date,
-        movementReason,
+        scheduledType,
       },
-      movementReasons: movementReasons
-        .map((values) => ({
-          value: values.code,
-          text: values.description,
-        }))
-        .sort(sortTextAlphabetically),
+      scheduledTypes,
       courtEvents,
       releaseEvents,
       transferEvents,
+      showCourtAppearances: !scheduledType || scheduledType === 'Court',
```

Comment:
Would it be possible to apply the scheduledType filter on the initial search `prisonApi.getTransfers` - that has booleans for each of them?!